rules/add-roles-from-sqlserver.md

```diff
@@ -12,14 +12,17 @@ This rule will query a SQL server database on each login and add a `roles` array
 
 ```js
 function (user, context, callback) {
-  getRoles(user.email, function(err, roles) {
-    if (err) return callback(err);
+  if(user.email && user.email_verified){
```

Comment:
Can we follow the same pattern we use in other rules (e.g.: account linking):

At the top of the rule:

```
  if (!user.email || !user.email_verified) {
    return callback(null, user, context);
  }
```

Reply:
It simplifies things and makes it more consistent (applies to all rules)

Reply:
Could you also please add tests for the `user.email_verified` property for the updated rules 🙏